modules/logging-loki-reliability-hardening.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-loki.adoc
+
+:_content-type: REFERENCE
+[id="logging-loki-reliability-hardening_{context}"]
+= Configuring Loki to tolerate node failure
+
+In logging 5.8 and newer versions, the Loki Operator supports setting pod anti-affinity rules to request that pods of the same component are scheduled on different available nodes in the cluster.
+
+include::snippets/about-pod-affinity.adoc[]
+
+[id="logging-loki-reliability-hardening-default-settings_{context}"]
+== Default Loki settings
+
+The Operator sets default, preferred `podAntiAffinity` rules for all Loki components, which includes the `compactor`, `distributor`, `gateway`, `indexGateway`, `ingester`, `querier`, `queryFrontend`, and `ruler` components.
+
+.Example default preferred rule settings for the ingester component
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: lokistack-dev
```

Comment:
Also, the docs currently about loki have the lokistack name as 'logging-loki' :)

Reply:
Updated, thanks